src/Base/ExceptionFactory.h

```diff
@@ -24,10 +24,9 @@
 #ifndef BASE_EXCEPTIONFACTORY_H
 #define BASE_EXCEPTIONFACTORY_H
 
-
-#include <Python.h>
-
 #include "Factory.h"
+#include "GeometryPyCXX.h"

```

Comment:
Thanks for reviewing.
This change was just an attempt to get rid of Python.h. Since i am no Pythionist, I could not judge if GeometryPyCXX.h loads eventually the same or less than Python.h. Knowing my limitations in this regards, i will revert the change and hand over a proper change to you.